cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1994,26 +1994,29 @@ def platform_dump(
         """Print informations about platforms currently defined.
         """
         if print_platform_names:
-            with suppress(ItemNotFoundError):
-                self.dump_platform_names(self)
+            self.dump_platform_names(self)
         if print_platforms:
-            with suppress(ItemNotFoundError):
-                self.dump_platform_details(self)
+            self.dump_platform_details(self)
 
     @staticmethod
     def dump_platform_names(cfg) -> None:
         """Print a list of defined platforms and groups.
         """
+        # Platforms is always defined with at least localhost
         platforms = '\n'.join(cfg.get(['platforms']).keys())
-        platform_groups = '\n'.join(cfg.get(['platform groups']).keys())
         print(f'{PLATFORM_REGEX_TEXT}\n\nPlatforms\n---------', file=stderr)
         print(platforms)
-        print('\n\nPlatform Groups\n--------------', file=stderr)
```

Comment:
FYI, this extra newline was probably present for compliance with ReST code style.

Reply:
Is this being used in the docs somewhere?

Reply:
It'd be pointless using it in the Cylc Docs, but I could imagine feeding it into metomi docs. I don't think that we do that at the moment, though.